src/components/table/TableItem.js

```diff
@@ -360,6 +359,8 @@ class TableItem extends PureComponent {
       colspan,
       viewId,
       keyProperty,
+      modalVisible,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['modalVisible' is missing in props validation (react/prop-types)](https://app.codacy.com/app/metasfresh/metasfresh-webui-frontend/pullRequest?prid=4666357)